dataverse-ext-common/src/main/java/edu/harvard/iq/dataverse/importer/metadata/ImporterFieldKey.java

```diff
@@ -0,0 +1,22 @@
+package edu.harvard.iq.dataverse.importer.metadata;
+
+/**
+ * This interface should be implemented by enums which will be used as a set of keys
+ * for elements of interface of importer form.
+ */
+public interface ImporterFieldKey {
+    /**
+     * This value is (and has to be) used by those elements of form that should
+     * not be processed.
+     */
+    ImporterFieldKey IRRELEVANT = () -> "IRRELEVANT";
+
+    /**
+     * The name used for id generation on view.
+     */
+    String getName();
+
+    default boolean isRelevant() {
+        return !IRRELEVANT.equals(this);
```

Comment:
Why not simply return `true` here and let the implementor override if needed? Having a default method and limiting it's use to a single value comparison seems counterintuitive. The `IRRELEVANT` object could simply return `false` from that method instead, so that the main interface would not have to rely on one of the implementations.

Reply:
good point